lua/nvim-treesitter/parsers.lua

```diff
@@ -2098,6 +2098,15 @@ list.terraform = {
   maintainers = { "@MichaHoffmann" },
 }
 
+list.tfvars = {
```

Comment:
Please sort these entries (in lockfile and readme as well) alphabetically.

Reply:
Done 🫡